src/data_api/orders.se

```diff
@@ -155,164 +155,189 @@ def getWorstBidOrderID(market: address, outcome: uint256):
 def getWorstAskOrderID(market: address, outcome: uint256):
     return(self.worstOrder[market][outcome].ask: address)
 
-# @private
-def updateBestBidOrder(orderID: address, market: address, fxpPrice, outcome: uint256, betterOrderID: uint256):
-    if(msg.sender != self):
-        ~invalid()
-    bestBidOrderID = self.bestOrder[market][outcome].bid
-    # If best bid is not set, this order is the new best bid
-    if(bestBidOrderID == 0):
-        self.bestOrder[market][outcome].bid = orderID
+# @public
+def isBetterPrice(type: uint256, fxpPrice, orderID: address):
```

Comment:
All unlabeled parameters are `int256` by default (which is why none of my code explicitly labels parameters as `int256`).  `fxpPrice` can be negative in scalar markets, which is why it is int256 rather than uint256.